ClientLibrary/Contract/TextRecognitionLine.cs

```diff
@@ -48,13 +48,21 @@ public class HandwritingTextLine
         /// </value>
         public int[] BoundingBox { get; set; }
```

Comment:
The `BoundingBox` and `Polygon` represent the same information.  This is confusing to the consumer.  Consider the following instead:
```
    public class TextRecognitionWord
    {
        /// <summary>
        /// Gets or sets the text.
        /// </summary>
        /// <value>
        /// The text.
        /// </value>
        public string Text { get; }

        /// <summary>
        /// Gets the polygon
        /// </summary>
        /// <value>
        /// The polygon
        /// </value>
        [JsonIgnore]
        public Polygon Polygon {get;}

        [JsonConstructor]
        public TextRecognitionWord(string text, int[] boundingBox)
        {
            Text = text;
            Polygon = Polygon.FromArray(boundingBox);
        }
    }

```

Reply:
Old MSOcr feature use similar format, so recommend to keep it.